rbac/ledger_sync/outbound/listener.py

```diff
@@ -44,10 +47,39 @@ def listener():
 
     finally:
         try:
-            database.disconnect()
+            conn.disconnect()
```

Comment:
Same here as above

Reply:
yes, there is a close function, i have changed it.